src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -748,6 +751,13 @@ type PandaproxyAPITLS struct {
 	// duplicate the secret to the same namespace as redpanda CRD to be able to
 	// mount it to the nodes
 	NodeSecretRef *corev1.ObjectReference `json:"nodeSecretRef,omitempty"`
+	// If provided and RequireClientAuth is true, the operator uses the certificate
+	// in this secret instead of issuing client certificates. The secret is expected to provide
+	// the following keys: 'ca.crt'.
+	// If ClientCACertRef points to a secret in a different namespace, the operator will
+	// duplicate the secret to the same namespace as the cluster CR to be able to
+	// mount it to the nodes.
+	ClientCACertRef *corev1.ObjectReference `json:"clientCACertRef,omitempty"`
```

Comment:
In the scenario that the CA cert is updated, will RP pick up the updated CA cert? Is it existing functionality?

Reply:
Yes! The server watches the configured cert and hot-reloads it. A restart is only needed if the cert path changes, but that shouldn't happen ever.